### ISSUE TITLE
First try of clean_ipynb

### ISSUE THREAD

**Issue by clstaudt** [KwatME/clean_ipynb#1]
```
Traceback (most recent call last):
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/termui.py", line 372, in style
    bits.append('\033[%dm' % (_ansi_colors.index(fg) + 30))
ValueError: tuple.index(x): x not in tuple

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/cls/anaconda3/bin/clean_ipynb", line 11, in <module>
    sys.exit(command_line_interface())
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/clean_ipynb/command_line_interface.py", line 27, in command_line_interface
    secho(ipynb_file_path, bg="black", fg="bright_green")
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/termui.py", line 420, in secho
    return echo(style(text, **styles), file=file, nl=nl, err=err, color=color)
  File "/Users/cls/anaconda3/lib/python3.7/site-packages/click/termui.py", line 374, in style
    raise TypeError('Unknown color %r' % fg)
TypeError: Unknown color 'bright_green'
```

**Comment by KwatME** [comment]
Hi @clstaudt,

Thank you very much for submitting the first issue :+1:  Very happy :smile: 

`clean_ipynb` prints out colorful messages to the console using `click`. And I think the cause of this error is having different `click` version, where my version supports using "bright_green" color.

To fix the issue, I'm going to stop coloring my prints messages to minimize the potential error such as this one.

I'm going to fix this and let you know.

Best,

**Comment by KwatME** [comment]
Hi @clstaudt,

I have released [0.0.4b](https://github.com/KwatME/clean_ipynb/releases/tag/0.0.4b) just now. This version specifies the required `click` version to be 7.0, the latest version that supports using more colors to style console messages. I hope this fixes your issue. If not, I will abandon using colors.

Please let me know if it works for you.

Best,

**Comment by clstaudt** [comment]
Yes, indeed the issue is fixed now. Thank you.
